amp-pwa-reader/src/server/server.js

```diff
@@ -20,10 +20,26 @@ const fs = require('fs');
 const path = require('path');
 const memCache = require('memory-cache');
 const pubBackend = require('./Backend.js');
+const enforce = require('express-sslify');
+const helmet = require('helmet');
+
+const ENVIRONMENT_PRODUCTION = 'production';
 
 const app = express();
 const pub = new pubBackend();
 
+app.use(helmet());
+
+if (app.get('env') === ENVIRONMENT_PRODUCTION) {
+    app.use((req, res, next) => {
+        if (req.path.startsWith('/tasks/')) {
```

Comment:
We have this `if` statement because we use the Tasks API from AppEngine for long running tasks. May be unnecessary if you are not running tasks.  If you are, [make sure that you are checking for the `X-AppEngine-Cron` header](https://github.com/GoogleChromeLabs/gulliver/blob/master/controllers/tasks.js#L25-L36), to ensure the URL can only be invoked by the AppEngine runtime.

Reply:
Thanks André! 
Hi @pbakaus, @morsssss, I'm not sure if there are any tasks on the Shadow Reader, do you think it's worth maintaining the if condition? 
Otherwise I can just remove that one.

Reply:
I'm not familiar with this Tasks API... I certainly haven't used it!  If @andreban hasn't used it in this instance, and I haven't either, I'd guess you could dump it.

Reply:
I haven't used tasks tasks. Should be safe to remove it.

Reply:
@andreban @morsssss just removed the condition and updated the PR. Latest version has been uploaded to steel-topic-208022.appspot.com.